src/Conseil.tsx

```diff
@@ -0,0 +1,75 @@
+import * as querystring from 'querystring';
+
+export interface ConseilFilter {
+    limit: number;
+    block_id: string[];
+    block_level: number[];
+    block_netid: string[];
+    block_protocol: string[];
+    operation_id: string[];
+    operation_source: string[];
+    account_id: string[];
+    account_manager: string[];
+    account_delegate: string[];
+}
+
+function queryNautilus(network: string, command: string, payload: string): Promise<string> {
+    return new Promise((resolve, reject) => {
+        let url = `https://localhost:1337/tezos/${network}/${command}`;
+        const xmlHttp = new XMLHttpRequest();
```

Comment:
[fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) is the request

Reply:
Opened #17 to do this as a separate ticket.